rfc/rfc004-distributed-document-format.md

```diff
@@ -0,0 +1,120 @@
+# RFC004 Distributed Document Format 
+
+|  |  |
+| :--- | :--- |
+| Nuts foundation | R.G. Krul |
+| Request for Comments: 004 | Nedap |
+|  | September 2020 |
+
+## Distributed Document Format
+
+### Abstract
+
+### Status of document
+
+This document is currently a draft.
+
+### Copyright Notice
+
+![](../.gitbook/assets/license.png)
+
+This document is released under the [Attribution-ShareAlike 4.0 International \(CC BY-SA 4.0\) license](https://creativecommons.org/licenses/by-sa/4.0/).
+
+## 1.  Introduction
+
+This RFC describes an interoperable, content agnostic data format for distributed networks which provides
+cryptographic authentication and verification of its contents, guaranteed ordering and access control.      
+
+## 2. Terminology
+
+* **Document**: piece of application data enveloped with metadata like signatures and references to other documents
+  published on a distributed network.
+
+Other terminology comes from the [Nuts Start Architecture](rfc001-nuts-start-architecture.md#nuts-start-architecture).
+
+## 3. Requirements
```

Comment:
This might have to move to the Nuts Start Architecture. It holds requirements for shared state.

Reply:
I'll try to find a good spot.